.clang-format

```diff
@@ -8,4 +8,4 @@ BinPackArguments: false
 BinPackParameters: false
 AllowShortFunctionsOnASingleLine: Empty
 IncludeBlocks: Preserve
-InsertBraces: true
+#InsertBraces: true
```

Comment:
Considered ok only due to https://github.com/soramitsu/kagome/blob/aa34f26ac559ac2c1afb8a4677e11ebf081e1d3a/CMakeLists.txt#L57